lib/ansible/modules/network/icx/icx_user.py

```diff
@@ -0,0 +1,394 @@
+#!/usr/bin/python
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+---
+module: icx_user
+version_added: "2.9"
+author: "Ruckus Wireless (@Commscope)"
+short_description: Manage the user accounts on Ruckus ICX 7000 series switches.
+description:
```

Comment:
Add a notes section for icx version tested against and link to platform guide
In the notes section, you can point to the icx platform guide https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/network/user_guide/platform_icx.rst

Refer: https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/network/junos/junos_interfaces.py#L107
Applicable for other icx modules as well.